src/renderer/OpenGL.zig

```diff
@@ -795,6 +833,27 @@ pub fn updateFrame(
             try self.prepKittyGraphics(state.terminal);
         }
 
+        if (self.current_background_image == null) {
+            if (self.background_image) |background_image| {
+                const img_path, const optional = switch (background_image) {
+                    .optional => |path| .{ path, true },
+                    .required => |path| .{ path, false },
+                };
```

Comment:
I think it's better to flip this logic and use `required` instead

```suggestion
                const img_path, const required = switch (background_image) {
                    .optional => |path| .{ path, false },
                    .required => |path| .{ path, true },
                };
```

Reply:
I've updated it, but just in case you are curious, we are using the other way around [here](https://github.com/ghostty-org/ghostty/blob/1ff91625981ef92add92624c01b6b9095c365ed1/src/config/Config.zig#L2829C1-L2832C11), [here](https://github.com/ghostty-org/ghostty/blob/1ff91625981ef92add92624c01b6b9095c365ed1/src/renderer/shadertoy.zig#L27C1-L30C11), or [here](https://github.com/ghostty-org/ghostty/blob/1ff91625981ef92add92624c01b6b9095c365ed1/src/apprt/gtk/App.zig#L1232C1-L1235C11). So, I had implemented it using `optional` as it was more common than the `required` implementation.